conf/development.ini

```
@@ -3,4 +3,5 @@ use = call:via.app:create_app
 
 nginx_server: http://localhost:9083
 client_embed_url: http://localhost:5000/embed.js
+internal_rewriter_url: http://localhost:9083/html/v
```

```suggestion
html_rewriter_url: http://localhost:9083/html/v
```

I think we should avoid "internal". Via 3 doesn't care whether the rewriter is part of Via 3 or not, it's just a URL that it redirects to. Also, I think the plan is for the new HTML rewriter to be external

---

It is specifically both:

 * Not the default
 * Not Via (which is also an HTML rewriter)

So I don't know what the right name is here, but I don't think this is it yet: `experimental_rewriter_url`?

---

I think the problem may be that we simply don't have a name for the new HTML rewriter. I guess we just need to come up with one. I feel like even just a random word would do. Or some hilarious play like viable or viaduct or vial (!) etc. The German word for via is uber. In Esperanto it's vojo. Etc.